### PR TITLE
fix: CSS inconsistencies between docs/tutorials/kubernetes-basics and…

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
ref: https://github.com/kubernetes/website/issues/34080

I have deleted
https://kubernetes.io/docs/tutorials/kubernetes-basics/public/css/styles.css
https://kubernetes.io/docs/tutorials/kubernetes-basics/public/css/overrides.css

and now CSS for docs/tutorials/kubernetes-basics is consistent with other pages.

### CODE REVIEW

**Comment by sftim** [comment]
This doesn't quite look right.
Compare the sidebar in https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/ to the [preview](https://deploy-preview-34188--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/scale/scale-intro/) of the new look.

We still need to style the elements of the sidebar. That might actually be the only part of the original CSS we need to keep.

I recommend adding that into a file within https://github.com/kubernetes/website/tree/main/assets/scss

**Comment by Shivansh-yadav13** [comment]
Squashing commit is causing 2 commits idk why

**Comment by sftim** [comment]
/label tide/merge-method-squash

OK to remove this label once this is _actually_ 1 commit

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+---
+title: Kubelet 凭据提供程序（CredentialProvider）（v1beta1）
+content_type: tool-reference
+package: credentialprovider.kubelet.k8s.io/v1beta1
+---
```
Comment (sftim) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1beta1.md:5]: Please omit this file from this PR.

File: assets/scss/_custom.scss [hunk 2]
```diff
@@ -853,3 +853,13 @@ div.alert > em.javascript-required {
...
+
+label
+{
+    font-weight: bold;
+
+    display: inline-block;
+
+    max-width: 100%;
+    margin-bottom: 5px;
+}
```
Comment (sftim) [inline assets/scss/_custom.scss:865]: We need to retain some styles; for example:
```suggestion
.content__box
{
    margin: 0 0 20px;
    padding: 20px;
}

.content__box_lined
{
    box-shadow: inset 0 0 0 1px #326de6;
}

.content__box_fill
{
    color: #fff;
    background: #326de6;
}
```

**Comment by sftim** [comment]
/hold

This PR includes a Chinese localized page. OK to unhold once that's fixed.

**Comment by Sea-n** [comment]
Hey @Shivansh-yadav13,

It's been a while since last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by divya-mohan0209** [comment]
Hey @Shivansh-yadav13 , please can you let us know if you're able to work on this PR?

File: content/en/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 3]
```diff
@@ -9,7 +9,6 @@
 
 <body>
 
-    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
 
 
 <div class="layout" id="top">
```
Comment (Sea-n) [inline content/en/docs/tutorials/kubernetes-basics/update/update-intro.html:14]: We can remove blank lines like this.

File: assets/scss/_custom.scss [hunk 4]
```diff
@@ -853,3 +853,18 @@ div.alert > em.javascript-required {
...
+.content__box_lined
+{
+    box-shadow: inset 0 0 0 1px #326de6;
+}
+.content__box_fill
+{
+    color: #fff;
+    background: #326de6;
+}
```
Comment (Sea-n) [inline assets/scss/_custom.scss:870]: Please add the ending newline (LF).

**Review by Sea-n — COMMENTED** [review]
Unhold since it's only touched SCSS and `content/en`.
/unhold

**Comment by sftim** [comment]
Thanks

/label tide/merge-method-squash
/approve

**Comment by sftim** [comment]
I reconsidered
/approve cancel

**Comment by sftim** [comment]
OK. I realized I hadn't checked how these pages looked for localizations other than English. The good news is: all OK.

Thanks again.

/label tide/merge-method-squash
/approve

**Pull request merged**